email-templates/task2.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>first email template</title>
+</head>
+<body style="font-family: Arial, sans-serif; text-align: center; background-color: #e4f3ea; margin: 20px auto; max-width: 600px; padding: 20px; border-radius: 8px; box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);">
+
+  <h1 style="color: #0074cc;">HAPPY NEW YEAR</h1>
+  <img src="hny.jpg" width="650" height="250">
+
+  <p style="color: #333;">
+    Dear Ronaldo,
+    
+  </p>
+
+  <p style="color: #333;">
+    Wishing you joy, success, and fulfillment in the coming year. May each day bring new opportunities and cherished moments. .
+  </p>
+
+  <p style="color: hsl(89, 51%, 46%);">
+    HAPPY NEW YEAR
+  </p>
+
+
+  <div style="margin-top: 20px; color: #888;">
+    <p>
+      your,<br>
+      Adithyan<br>
+      7012260905<br>
+      adithyanm007@gmail.com
+    </p>
+  </div>
+  <hr>
+  <a href="https://www.flipkart.com/jbl-tune-760nc-active-noise-cancelling-50-hr-playtime-fast-pair-multi-connect-bluetooth-headset/p/itm928f4dd795417?pid=ACCG74FYGFN9SZXP&lid=LSTACCG74FYGFN9SZXPCZUZZ4&marketplace=FLIPKART&q=head+phone+jbl&store=0pm%2Ffcn&srno=s_1_17&otracker=search&otracker1=search&fm=Search&iid=a3850019-1554-45d3-9908-cd0338e88c15.ACCG74FYGFN9SZXP.SEARCH&ppt=sp&ppn=sp&ssid=hgsskqnxuo0000001704266841924&qH=b12dde0e0ad2397d" style="color: #fff; text-decoration: none;">
+    <button>Click here to see my New Year gift</button>
+  </a>
```

Comment:
Don't use `button` inside `a` tag. Instead, you can use CSS to make the `a` tag like a button